sdks/python/apache_beam/io/fileio.py

```diff
@@ -463,13 +437,9 @@ def open(path,
            compression_type=CompressionTypes.AUTO):
     if compression_type == CompressionTypes.AUTO:
       compression_type = CompressionTypes.detect_compression_type(path)
-    elif compression_type == 'AUTO':
-      # TODO: Remove this backwards-compatibility soon.
```

Comment:
How have we determined that it's now safe to act on these TODOs? We should at least highlight these changes in release notes somewhere, right?